tsp_simple.cc

```diff
@@ -628,7 +628,7 @@ void RelationBuilder(const TSPTWDataDT& data, RoutingModel& routing,
       }
       break;
     case VehicleGroupDuration:
-      if (relation.lapse > -1) {
+      if (relation.lapse > 0) {
```

Comment:
What if we want 0 lapse in between, does 0-lapse make sense? :thinking: 

I don't know much about this parameter so if the question doesn't make sense ignore it

Reply:
It is not in between, this means that sum of those vehicles duration should be zero. This makes no sense for me, it corresponds to not providing those vehicles.

Reply:
You are right, that would make sense to no one.. 

I just missed the just huge `VehicleGroupDuration` above and jump to the "delay" definition between trips :facepalm:

Reply:
Just realized why I was mistaken because in the diff screen the modification look very "close" but they are two different cases in fact. Oh well, no harm done :) 

Why do we call this parameter "lapse" by the way, it should clearly be called "duration", no? It would match the relation `VehicleGroupDuration` too.